apps/system/test/marionette/notification_actions_test.js

```diff
@@ -58,4 +68,25 @@ marionette('launch an app via notification click', function() {
     });
   });
 
+  test('swiping to dismiss', function() {
+    // show utility tray
+    utilityTray.open();
+    utilityTray.waitForOpened();
+
+    // make sure we have our notification to click
+    notificationList.refresh();
+    var notifications = notificationList.getForApp(TARGET_APP_MANIFEST);
+    assert.equal(notifications.length, 1);
+
+    actions.flick(utilityTray.firstNotification, 45, 15, 90, 15, 30).perform();
+
+    // email will clear the notification from the tray, so wait
```

Comment:
Won't the notification be cleared by the swipe that was just performed, rather than by email?